docs/README.md

```diff
@@ -0,0 +1,23 @@
+# chisel3/docs/README.md
+
+This directory contains documentation on the code within this repository.
+Documents can either be written directly in markdown, or
+use embedded [mdoc](https://scalameta.org/mdoc/)
+which compiles against the `chisel3` (and dependencies) codebase
+as part of the PR CI checks,
+forcing the documentation to remain current with the codebase.
+The `src` folder contains the source from which these are generated.
+
+For each topic, we categorize documentation as described in
+[Divio's documentation system](https://documentation.divio.com/).
+
```

Comment:
see other comment. not sure if we should put this here, unless this is 100% the direction we are going in? (maybe it is)